tests/src/index.test.js

```diff
@@ -131,77 +129,65 @@ describe('#getToken', () => {
 
 describe('#uploadFile', () => {
   it('throws an error with no brand ID', () => {
-    bynder.uploadFile({
+    return expect(bynder.uploadFile({
```

Comment:
To make sure that jest waits for the promise to complete we need to either return the promise from the test, or await it. Also by the `expect(...).rejects` pattern we ensure that the test fails if the promise were to resolve.